File: AtlasWeb/Core/Modules/Tibbi_Surec/Patoloji_Modulu/PathologyTestRequestInfoFormViewModel.ts
//$B072D73D
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { PathologyTestProcedure } from "NebulaClient/Model/AtlasClientModel";
import { Type } from "NebulaClient/ClassTransformer";

export class PathologyTestRequestInfoFormViewModel extends BaseViewModel {
  @Type(() => PathologyTestProcedure)
  public _PathologyTestProcedure: PathologyTestProcedure = new PathologyTestProcedure();
  public ProcedureName: string;
  public textDescription: string;
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Patoloji_Modulu/PathologyTestRequestInfoFormViewModel.cs
//$C965F221
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;

namespace Core.Controllers
{
    public partial class PathologyTestProcedureServiceController
    {
    }
}

namespace Core.Models
{
    public partial class PathologyTestRequestInfoFormViewModel
    {
        public string ProcedureName
        {
            get;
            set;
        }

        public string textDescription
        {
            get;
            set;
        }
    }
}
